deploy_slave.sh

```diff
@@ -78,8 +78,7 @@ getent group piwheels || groupadd piwheels
 getent passwd piwheels || useradd -g piwheels -m -s /bin/bash piwheels
 passwd -d piwheels
 
-curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
-cat .cargo/env >> /home/piwheels/.bashrc
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sudo -u piwheels sh -s -- -y
```

Comment:
consider [`runuser`](https://man7.org/linux/man-pages/man1/runuser.1.html), which does not require the installation of `sudo` package and has generally simpler semantics

Reply:
I'm not too familiar with runuser, e.g. whether it fully clears the environment like `sudo` or keeps it by default like `su`? Also it seems to use PAM, not sure whether this is fully available in the build environment?

Reply:
`runuser` has semantics very similar to `su`, since it was originally derived from `su`. It keeps the existing environment by default, but some values will be overwritten as-if the target user were logging in.

The primary difference from `su` is that `runuser` never prompts for a password. It also ensures that the PAM session is set up, like the user were logging in, which makes it more likely to have the correct paths for installing user-local tools.

Admittedly, `runuser` is newer to me as well, but I've been trying to get away from `sudo` for simple use cases, since it's such a large and complex package that is often overkill. Also, piwheels is newer for me as well, so if something about the environment might preclude this, then I'll obviously defer to what is working ;)

Reply:
This su-like syntax works (while `-u` + `-l` "are mutually exclusive"):
```suggestion
curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | runuser -- - piwheels -s -- -y
```
- `--` ends `runuser` options, otherwise `-y` is interpreted as such.
- `-` defines a login shell, so the parent root user's environment is not passed through.
- `-s -- -y` arguments passed to the shell, second `--` here so that `-y` is not interpreted as shell option but as positional argument for the script.

@masenf
Since I have no experience with `runuser`, any objections from your end, before I push this change?

Reply:
@MichaIng looks good to me! thanks for fixing this up